### ISSUE TITLE
AnalyticalTable: Missing tooltip on sort icon

### ISSUE THREAD

**Issue by NaveenKarippai** [SAP/ui5-webcomponents-react#4172]
**Describe the bug**
The sort icon on the title of the table is not showing a tooltip on hover.

**To Reproduce**

1. Click on 'name' (any title) on the table
2. Choose 'Sort Ascending'
3. The sort icon appears on the title next to the "Name"
4. Hover on the sort icon

**Expected behavior**
Hovering on the sort icon should display a tooltip (useful for users with disability)

**Screenshots**
<img width="1509" alt="Screenshot 2023-02-17 at 12 19 24" src="https://user-images.githubusercontent.com/11843784/219652001-86cf3a1c-ea89-4c45-8477-92c1a74069e7.png">

**UI5 Web Components for React Information**
version: 1.7.0, 1.8.1
Operating System: Macbook (checked on Windows as well)
Browser: Google Chrome

**Comment by Lukas742** [comment]
Hi @NaveenKarippai 

thanks for bringing this up. The icons are purely decorative and shouldn't be noticed by screen readers, that's why it's not possible to focus them for example. That's why we don't offer a tooltip there.
Still, we were lacking screen reader support, because we didn't announce if a column was filtered, sorted or grouped. The linked PR will fix this.
